components/director/internal/open_resource_discovery/validation.go

```diff
@@ -1030,6 +1036,30 @@ func validateAPIPartOfConsumptionBundles(value interface{}, targetURLs json.RawM
 	return nil
 }
 
+func validateDefaultConsumptionBundle(value interface{}, partOfConsumptionBundles []*model.ConsumptionBundleReference) error {
+	defaultConsumptionBundle, ok := value.(*string)
+	if !ok {
+		return errors.New("error while casting to defaultConsumptionBundle")
+	}
+
+	if defaultConsumptionBundle == nil {
```

Comment:
The field is optional by the spec that's why we store it as a pointer in our codebase. If it isn't provided in the document the field will be nil which is fine. 

If it is provided but is an empty string, that will be caught by the validation below because the empty string will not be part of the array `partOfConsumptionBundles` (i.e there is no bundle with ID empty string so your defaultBundle can not be an empty one).